substrate/client/transaction-pool/tests/zombienet/network-specs/asset-hub-high-pool-limit-fatp.toml

```diff
@@ -0,0 +1,66 @@
+[settings]
+timeout = 1500
+
+[relaychain]
+default_image = "parity/polkadot:latest"
+default_command = "polkadot"
+chain = "rococo-local"
+
+[[relaychain.nodes]]
+name = "alice"
+rpc_port = 9944
+validator = true
+
+[[relaychain.nodes]]
+name = "bob"
+rpc_port = 9945
+validator = true
+
+[[parachains]]
+id = 2000
+chain = "asset-hub-rococo-local"
+default_command = "polkadot-parachain"
+default_image = "parity/polkadot-parachain:latest"
+cumulus_based = true
+default_args = [
+	"--force-authoring",
+	"--pool-kbytes 2048000",
+	"--pool-limit 500000",
+	"--pool-type=fork-aware",
+	"--rpc-max-connections 15000",
+	"--rpc-max-response-size 150",
+	"--rpc-max-subscriptions-per-connection=128000",
+	"--state-pruning=1024",
+	"-laura::cumulus=info",
+	"-lbasic-authorship=info",
+	"-lpeerset=info",
+	"-lsub-libp2p=info",
+	"-lsync=info",
+	"-ltxpool=debug",
+]
+[parachains.genesis.runtimeGenesis.patch.balances]
+devAccounts = [
+	100,
```

Comment:
we could have `20_000` here, but let's bump it in follow-up (also other tomls).

Reply:
Generating 20k accounts takes a while, so testing will take longer. I think it is best to add 20k accounts only if the scenarios will make use of them.